config/locales/forms/question_text.yml

```
@@ -0,0 +1,12 @@
+en:
+  helpers:
+    label:
+      forms_question_text_form:
+        question_text: What's your question?
```

I think this should be a curly apostrophe?

```suggestion
        question_text: What’s your question?
```